src/test/java/uk/gov/hmcts/reform/civil/service/citizenui/DashboardClaimInfoServiceTest.java

```diff
@@ -164,7 +200,21 @@ void shouldIncludePaymentDateWhenItExists() {
             "authorisation",
             "123"
         );
-        assertThat(claimsForDefendant.size()).isEqualTo(2);
+        assertThat(claimsForDefendant.size()).isEqualTo(3);
         assertThat(claimsForDefendant.get(1).getPaymentDate()).isEqualTo(now.toLocalDate());
     }
+
+    @Test
```

Comment:
Can you add a test for when the query returns 0 cases? Also add one where the query returns greater than the page size